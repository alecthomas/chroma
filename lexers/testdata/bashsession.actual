$ echo "Hello, world!"
Hello, world!
%	pwd
/User/foo
>  make -j build
%ls | wc -l
5
