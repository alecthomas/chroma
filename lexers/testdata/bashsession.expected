[
  {"type":"GenericPrompt","value":"$ "},
  {"type":"NameBuiltin","value":"echo"},
  {"type":"Text","value":" "},
  {"type":"LiteralStringDouble","value":"\"Hello, world!\""},
  {"type":"Text","value":"\n"},
  {"type":"GenericOutput","value":"Hello, world!\n"},
  {"type":"GenericPrompt","value":"%\t"},
  {"type":"NameBuiltin","value":"pwd"},
  {"type":"Text","value":"\n"},
  {"type":"GenericOutput","value":"/User/foo\n"},
  {"type":"GenericPrompt","value":">  "},
  {"type":"Text","value":"make -j build\n"},
  {"type":"GenericPrompt","value":"%"},
  {"type":"Text","value":"ls "},
  {"type":"Punctuation","value":"|"},
  {"type":"Text","value":" wc -l\n"},
  {"type":"GenericOutput","value":"5\n"}
]
